### PR TITLE
Initial CLI API integration tests

### PR DESCRIPTION
**Pull request by iketheadore**

### CODE REVIEW

File: src/api/cli/integration/status.golden [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "running": true,
+    "num_of_blocks": 181,
```
Comment (gz-c) [inline src/api/cli/integration/status.golden]: number of blocks is 181 now?

File: src/api/cli/integration/integration_test.go [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+	defer f.Close()
+
+	err = json.NewDecoder(f).Decode(obj)
+	require.NoError(t, err)
+}
+
+func nodeAddress() string {
+	addr := os.Getenv("SKYCOIN_NODE_HOST")
+	if addr == "" {
+		return "http://127.0.0.1:6420"
```
Comment (gz-c) [inline src/api/cli/integration/integration_test.go]: this is http api interface address not rpc address

File: src/api/cli/integration/integration_test.go [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+	wltPath := filepath.Join(walletDir, walletName)
+	var w wallet.ReadableWallet
+	loadJSON(t, wltPath, &w)
+
+	var expect wallet.ReadableWallet
+	loadJSON(t, "generateAddresses.golden", &expect)
+	require.Equal(t, expect, w)
+}
+
+func TestStabeStatus(t *testing.T) {
```
Comment (gz-c) [inline src/api/cli/integration/integration_test.go]: `Stable`

File: src/api/cli/integration/status.golden (hunk 1 above)
Comment (iketheadore) [reply src/api/cli/integration/status.golden]: Yes, I think so. The block of seq 180 is exist in the db.

**Comment by gz-c** [comment]
merge conflict

File: src/api/cli/integration/integration_test.go [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+
+var (
+	binaryPath string
+	walletDir  string
+)
+
+var update = flag.Bool("update", false, "update golden files")
+
+func TestGenerateAddresses(t *testing.T) {
+	if !doLive(t) && !doStable(t) {
```
Comment (gz-c) [inline src/api/cli/integration/integration_test.go]: Need to change this to `doLiveOrStable` like https://github.com/skycoin/skycoin/pull/989

File: src/api/cli/integration/integration_test.go [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+		t.Run(fmt.Sprintf("%v", txid), func(t *testing.T) {
+			var tx webrpc.TxnResult
+			output, err := exec.Command(binaryPath, "transaction", txid).CombinedOutput()
+			require.NoError(t, err)
+			require.NoError(t, json.NewDecoder(bytes.NewReader(output)).Decode(&tx))
+			require.Equal(t, txid, tx.Transaction.Transaction.Hash)
+		})
+	}
+}
+
```
Comment (gz-c) [inline src/api/cli/integration/integration_test.go]: What about TestLiveTransaction

**Pull request merged**